components/composer/CHANGELOG.md

```diff
@@ -1,6 +1,4 @@
-# Pending Changes
```

Comment:
Would you mind adding a blank template to the top of these changelog files for the next person?
```
# Pending Changes

## Breaking Changes
## New Features
## Bug Fixes
```

Reply:
Done 👍🏽